src/pages/calendar/components/Requisites.tsx

```diff
@@ -25,70 +25,67 @@ export function DisplayRequirement(
     // If there's a quantity, then we have a list of requisites to display
   } else if ('quantity' in req) {
     //Then it is a NestedPreCoRequisite
-    const reqs = nestedReq(req);
+    const reqs = req.reqList ? req.reqList : [];
```

Comment:
```suggestion
    const reqs = req.reqList ?? [];
```
Just a more clean way to do these cases of ternaries where the condition is the variable being assigned https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing